case-studies/policy/README.md

```diff
@@ -85,17 +85,23 @@ sudo apy-get install ipset
 
 # 2. Setup environment or .env file
 # api-server address
-API_SERVER_ADDRESS="http://localhost:8080"
+API_SERVER_ADDRESS="address of api-server"
 # ip address of api-server
-API_SERVER_IPV4="127.0.0.1"
+API_SERVER_IPV4="ipv4 of api-server"
```

Comment:
What is the different between this one and API_SERVER_ADDRESS?

Reply:
API_SERVER_ADDRESS for agent call API. API_SERVER_ADDRESS is domain name or ip with port
API_SERVER_IPV4 is ip of api-server. Use for iptables.
I have add more comments